SU2_CFD/include/solver_structure.hpp

```diff
@@ -466,6 +475,64 @@ class CSolver {
    */
   su2double* GetPoint_Max_Coord_BGS(unsigned short val_var);
   
+  /*!
```

Comment:
Could this methods and associated variables be put in CVerificationSolution? The CSolver namespace is very cluttered and with this naming these variables and methods are asking for trouble.

Reply:
I suppose you mean the methods AddError_RMS, AddError_Max, SetError_RMS and SetError_Max? Yes, I think we can move those to CVerificationSolution, but I am not 100 percent sure. @economon, is there a compelling reason to keep these methods in CSolver?

Reply:
Yes, this entire new section line 478 to 535

Reply:
Yep, I like this idea. I think that the Set/Add/GetError_*() can all move to CVerificationSolution, as well as the SetVerificationError() routine, which just does a final communication and stores the value.

Prob need to keep the ComputeVerificationError() in the solvers though, since this routine holds the loop over the DOFs to compute the local error, and these loops have different structures in the different solver (FV vs DG, for instance).

Reply:
I moved these methods to CVerificationSolution. Indeed, this makes a lot more sense. @pcarruscag, thanks for the suggestion.